CHANGELOG.md

```diff
@@ -1,3 +1,151 @@
+## 1.14.0 (2019-08-21)
+
+This feature release adds a read-only mode to the Flux daemon, adds
+support for mapping images in `HelmRelease` resources using YAML dot
+notation annotations, eases the deployment of Flux with a new `fluxctl
+install` command which generates the required YAML manifests, lots of
+documentation improvements, and many more.
+
+### Fixes
+
+- Fetch before branch check to detect upstream changes made after the
+  initial clone
+  [fluxcd/flux#2371][]
+
+### Enhancements
+
+- With `--git-readonly`, `fluxd` can now sync a git repo without having
+  write access to it. In this mode, `fluxd` will not make any commits
+  to the repo.
+  [fluxcd/flux#1807][]
+- Mapping images in `HelmRelease resources` using YAML dot notation
+  annotations is now supported
+  [fluxcd/flux#2249][]
+- `fluxctl` has a new `install` command to ease generating the YAML
+  manifests required to deploy Flux
+  [fluxcd/flux#2287][]
+- Kubectl and Kustomize have been upgraded
+  - `kubectl` -> `1.13.8` [fluxcd/flux#2327][]
+  - `kustomize` -> `3.1.0` [fluxcd/flux#2299][]
+- The annotation domain has been changed to `fluxcd.io`, but backwards
+  compatibility with the old (`flux.weave.works`) domain is maintained
+  [fluxcd/flux#2219][]
+- The amount of sorts done by `ListImagesWithOptions` has been reduced
```

Comment:
s/amount/number/

Reply:
(amount is usually used only for continuous quantities, e.g., "that is a large amount of water")